deno/lib/locales/en.ts

```diff
@@ -70,11 +70,11 @@ const errorMap: ZodErrorMap = (issue, _ctx) => {
       if (issue.type === "array")
         message = `Array must contain ${
           issue.exact ? "exactly" : issue.inclusive ? `at least` : `more than`
-        } ${issue.minimum} element(s)`;
+        } ${issue.minimum} ${issue.minimum <= 1 ? "element" : "elements"}`;
```

Comment:
this is overkill, maybe do this instead:

```ts
message = `Array must contain ${
          issue.exact ? "exactly" : issue.inclusive ? `at least` : `more than`
        } ${issue.minimum} element${issue.minimum > 1 ? "s" : ""}`;
```

Reply:
I have made the changes. Please have a look and let me know if any further changes are required.

Reply:
Looks better, it's up to maintainers to approve and merge this PR.